events/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.contrib.gis.db import models as model
+
+
+class Category(models.Model):
+    category_name = models.CharField(max_length=255)
+    
+    parent = models.ForeignKey(
+        "events.Category", 
+        related_name="categories", 
+        null=True, 
+        blank=True, 
+        on_delete=models.CASCADE
+    )
+
+
+class Event(models.Model):
+    class EventType(models.TextChoices):
+        onsite = 'ONSITE', 'Onsite'
+        online = 'ONLINE', 'Online'
+        hybrid = 'HYBRID', 'Hybrid'
+
+    event_type = models.CharField(
+        max_length=10,
+        choices=EventType.choices,
+        default=EventType.onsite,
+    )
+    name = models.CharField(max_length=32)
+
+    registration_start_time = models.DateTimeField()
+    registration_end_time = models.DateTimeField()
+
```

Comment:
why so many empty lines

Reply:
I did this to separate model fields and divide them in sections with same field for better readability purpose.